python/cugraph/cugraph/experimental/dask/__init__.py

```diff
@@ -13,7 +13,11 @@
 
 from cugraph.utilities.api_tools import experimental_warning_wrapper
 
+from cugraph.dask.sampling.uniform_neighbor_sample import \
+    uniform_neighbor_sample
+uniform_neighbor_sample = \
+    experimental_warning_wrapper(uniform_neighbor_sample)
```

Comment:
We can immediately promote this to stable too, which means we'll be replacing the current MG neighborhood sampling algo with this one (a breaking change).  We think this is okay though, since this was mainly added for DGL and the DGL effort is just starting to use cugraph MG neighbor sampling (IOW, we don't think this will actually break anyone).
Of course, since the old version was released in 22.04, there's no way to guarantee a community member isn't using it, but we still think the chance is small enough not to warrant the extra expense of maintaining two versions to support our normal deprecation process.  I'll add a "breaking" label to this PR though too.

Reply:
I concur

Reply:
I agree. It didn't make it into the 22.04 release html docs, so a community member who used it had to have followed development closely.